HCCGo/main.js

```diff
@@ -29,5 +29,6 @@ app.on('ready', function() {
 
     mainWindow.once('ready-to-show', () => {
         mainWindow.show();
+        //mainWindow.webContents.openDevTools();
```

Comment:
Simple way to turn on/off debugging tools for packaged applications. Release versions has this line commented, Debug versions has this line uncommented.